about.html

```diff
@@ -0,0 +1,139 @@
+         heads of state, and the founders and CEOs of technology companies and fast-growing startups, to ask a simple 
+         question: Where to next?<br><br>
+
+        See you in Lisbon this November.
+    </p></div>
+    <div class="datediv-about">
+        <p class="locationpara-about">
+            Please contact us per Email for further questions about Web Summit 2023!
+        </p>
+        <h3 class="dateheading-about underline">baqar2023@webSummitProject.org</h3>
+        
+      </div>
+</div>
+</section>
+<!--*******Desktop Headline Section******-->
+<div class="headlinedesktop-about">
+    <div class="topdesktop-about">
+        <h3 class="topheadingdesktop-about">"Hello Tech World!"</h3>
+    </div>
+    <div class="topheadingdesktopdiv-about">
+        <h1 class="topheadingh1desktop-about">Web Global Annual <br> Summit  2023</h1>
+        <div class="paradivdesktop-about"><p class="headlineparadesktop-about">What is Web Summit? <br><br>
+            Politico has said we run “the world’s premier tech conference”,
+            the Atlantic that Web Summit is “where the future goes to be born”, and the New York Times that we assemble “a grand 
+            conclave of the tech industry’s high priests”. <br>    
+            At a time of great uncertainty for many industries and, indeed, the world itself, we gather policymakers,
+             heads of state, and the founders and CEOs of technology companies and fast-growing startups, to ask a simple 
+             question: Where to next?<br><br>
+    
+            See you in Lisbon this November.
+        </p></div>
+        <div class="datedivdesktop-about">
+            <p class="locationparadesktop-about">
+                Please contact us per Email for further questions about Web Summit 2023!
+            </p>
+            <h3 class="dateheadingdesktop-about underline">baqar2023@webSummitProject.org</h3>
+            
+          </div>
+    </div>
+    </div>
+    </article>
+    <section class="logo-section">
+<h2 class="logosection-heading">Web Global Annual Summit 2023 Logo</h2>
+<hr class="hr"/>
+<p class="logosection-para">Since 2014, both versions of the logo make use of Rui Abreu's Azo Sans.  It all began with a design brief.
+     Designers across the globe delivered design magic. This Web Summit vector logo is 100% vector-based.</p>
+     <div class="logo-div">
+        <img class="logo-specifics" src="./images/websummitlogo.png" alt="summit logo">
+     </div>
+    </section>
+    <section class="pastproject-section">
+        <h2 class="logosection-heading">See the Past Global Summits</h2>
+        <hr class="hr"/>
+        <p class="logosection-para">Take a look at the last two Web Summits which took place. Web Summit 2022 and 2020 </p>
+       <div class="pastproject-images">
+        <img class="summit22" src="./images/websummit2022.jpg" alt="Summit">
+        <img class="summit20" src="./images/websummit2020.jpg" alt="Summit">
+    </div>
```

Comment:
- [ OPTIONAL]  consider adding the blur background to your pictures to match to the one from the provided design.

Reply:
All the optional changes I will incorporate them later on. Please consider the time factor I also have coding challenges to complete for this week. This is my fourth review that I am sending again for a review please approve